DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/partner-account-control/affiliate-documentation.md

If a merchant is unable to find and answer in the above links, they can always search for topics using the [search bar]( https://docs.multisafepay.com) on our MultiSafepay Documentation Center page. 

> For further inquiries, affiliates can contact us at <partner@multisafepay.com>

Reviewer: ```suggestion
For any questions, affiliates can email <partner@multisafepay.com>
```
